starter-code/pizza.js

```diff
@@ -33,53 +48,120 @@ function renderEverything() {
 }
 
 function renderPepperonni() {
-  document.querySelectorAll('.pep').forEach(function($pep){
+  document.querySelectorAll('.pep').forEach(function ($pep) {
     if (state.pepperonni) {
       $pep.style.visibility = "visible";
-    }
-    else {
+    } else {
       $pep.style.visibility = "hidden";
     }
   })
 }
 
 function renderMushrooms() {
-  // Iteration 1: set the visibility of `<section class="mushroom">`
+  document.querySelectorAll('.mushroom').forEach(function ($mush) {
+    if (state.mushrooms) {
+      $mush.style.visibility = "visible";
+    } else {
+      $mush.style.visibility = "hidden";
+    }
+  })
 }
 
 function renderGreenPeppers() {
-  // Iteration 1: set the visibility of `<section class="green-pepper">`
+  document.querySelectorAll('.green-pepper').forEach(function ($green) {
+    if (state.greenPeppers) {
+      $green.style.visibility = "visible";
+    } else {
+      $green.style.visibility = "hidden";
+    }
+  })
 }
 
+
 function renderWhiteSauce() {
-  // Iteration 2: add/remove the class "sauce-white" of `<section class="sauce">`
+  document.querySelectorAll('.sauce').forEach(function ($sauce) {
+    if (state.whiteSauce) {
+      $sauce.style.visibility = "visible";
+    } else {
+      $sauce.style.visibility = "hidden";
+    }
+  })
 }
 
 function renderGlutenFreeCrust() {
-  // Iteration 2: add/remove the class "crust-gluten-free" of `<section class="crust">`
+  document.querySelectorAll('.crust').forEach(function ($crust) {
```

Comment:
See my comment above. Also here you want to add/remove a class. Not make something hidden or visible.